src/components/InscriptionSignUp.jsx

```diff
@@ -38,8 +38,10 @@ export default function useForm(validateInfo) {
     delete values.password2;
     api.post("/auth/signup", values).then((res) => {
       setValues(res.data);
-      dispatch({ type: "SET_ID", setId: res.data.idClient });
-      dispatch({ type: "SET_INFOS", setInfos: res.data.infosClients });
+      dispatch({ type: "SET_ID", test: res.data.idClient });
+      dispatch({ type: "SET_INFOS", test: res.data.infosClients });
+      dispatch({ type: "SET_INFOS", test: res.data.tokenClient });
```

Comment:
- "test" is a bad key name
- You dispatch the same action twice in a row, it doesn't seem like a good idea